pkgs/development/compilers/llvm/common/compiler-rt/default.nix

```diff
@@ -188,6 +188,9 @@ stdenv.mkDerivation ({
     substituteInPlace ../libcxx/utils/merge_archives.py \
       --replace-fail "import distutils.spawn" "from shutil import which as find_executable" \
       --replace-fail "distutils.spawn." ""
+  '' + lib.optionalString (lib.versionAtLeast release_version "19") ''
+    substituteInPlace cmake/Modules/AddCompilerRT.cmake \
+      --replace-fail 'find_program(CODESIGN codesign)' ""
```

Comment:
Might be nice to have a comment about the unsupported option it tries to pass so that we know to drop this when e.g. we get an `rcodesign`‐based thing that can do it correctly.

Reply:
I would like to explore making Darwin `useLLVM` for the LLVM bootstrap after 25.05. If the linker is LLD, it won’t try to invoke `codesign`. In the interim, this is fine.